### DISCUSSION TITLE
測試2

### DISCUSSION THREAD

**Discussion by edaw1111** [1101-GS3073-C/Python-Community-GS3073-C#5] [Q&A] [answered]
貼文測試2

**Comment by edaw1111** [comment, marked as answer]
留言測試2

**Reply by chia-kai-chang** [reply]
> 11111

Good job 👏

![image](https://user-images.githubusercontent.com/1768891/133051707-5fcd84bc-b7f3-4a09-85b1-2be5c196745f.png)

![image](https://user-images.githubusercontent.com/1768891/133051883-196b4b51-3e79-4d3f-8357-b377b0b5ada1.png)

用ipad 張貼文章也可以喔😌
